spec/services/teacher_training_public_api/sync_all_providers_and_courses_spec.rb

```diff
@@ -44,7 +44,7 @@
 
```

Comment:
Could we add a test here that checks we correctly filter providers based on when they were updated?

Reply:
I think this is done via lines 52 - 55?

```
stub_teacher_training_api_providers(
          recruitment_cycle_year: recruitment_cycle_year,
          filter_option: { 'filter[updated_since]' => updated_since },
        )
```

Reply:
Ah I see 👍